__mocks__/fs.cts

```diff
@@ -0,0 +1,3 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { fs } = require("memfs");
+module.exports = fs;
```

Comment:
Also removed mock-fs package in favour of still-maintained memfs.

The `__mocks__` directory is supported by [vitest](https://vitest.dev/guide/mocking#file-system) (and [Jest](https://jestjs.io/docs/manual-mocks)) framework.